examples/getting-started-movielens/04-Triton-Inference-with-HugeCTR.ipynb

```diff
@@ -55,29 +55,42 @@
    "id": "71304a10",
```

Comment:
<p>you can remove <code>After you started the container&nbsp;</code>.Just say <code> We can launch the Triton Inference Server with the following command:</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>